storage-resize-images/functions/__tests__/emulator/emulator-params.env

```diff
@@ -0,0 +1,13 @@
+LOCATION=us-central1
+DATABASE_INSTANCE=extensions-testing
+IMG_BUCKET=default-bucket
+CACHE_CONTROL_HEADER=false
+IMG_SIZES=200x200
+RESIZED_IMAGES_PATH=resized
+DELETE_ORIGINAL_FILE=false
+IMAGE_TYPE=jpeg,webp,png,tiff,false
+INCLUDE_PATH_LIST=/projects
+FIREBASE_STORAGE_EMULATOR_HOST=localhost:9199
+# need to be added as otherwise it is making loop of death in local environment emulator
```

Comment:
If this is an issue with the emulator can we create a separate issue in https://github.com/firebase/firebase-tools and add the comment here with the description similar to...

```
local emulation causes duplicate file resizing, see more via issue: #
```